src/components/charts/glucose/glucose-distribution.tsx

```
@@ -63,4 +80,27 @@ export const GlucoseDistributionChart = ({ glucoseLogs, chartConfig }: { glucose
+const CustomizedDot: React.FC<CustomizedDotProps> = (props) => {
+    const { cx, cy, payload, targetLow, targetHigh } = props;
+
+    if (payload === undefined) return null
+
+    const glucoseValue = payload.glucose;
+
+    if (typeof cx !== 'number' || typeof cy !== 'number') {
+        return null;
+    }
+
+    const fillColor = useMemo(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: payload

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)